CannibalisticZombies/Assets/PlayerCharacterController.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerCharacterController : MonoBehaviour
+{
+    public float moveSpeed;
+
+    public Transform orientation;
+
+    float horizontalInput;
+    float verticalInput;
+
+    Vector3 moveDir;
+
+    Rigidbody rb;
+
+    private void Start()
+    {
+        rb = GetComponent<Rigidbody>();
+        rb.freezeRotation = true;
+    }
+
+    private void Update()
+    {
+        PInput();
+    }
+
+    private void FixedUpdate()
+    {
+        Move();
+    }
+    void PInput()
+    {
+        horizontalInput = Input.GetAxisRaw("Horizontal");
+        verticalInput = Input.GetAxisRaw("Vertical");
+    }
+
+    void Move()
+    {
+        moveDir = orientation.forward * verticalInput + orientation.right * horizontalInput;
+
+        rb.AddForce(moveDir.normalized * moveSpeed * 10f,ForceMode.Force);
```

Comment:
I tested the movement, and the player is rapidly speeding up. I am not familiar with using AddForce for movement, so I am not sure what the fix would be for this. Instead, I suggest setting rb.velocity similar to how you did in Punchline, but instead in 3D.

Reply:
Additional comment: 
Put the player scripts into the folder Assets/Scripts/Character/Player